<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputwindow</class>
 <widget class="QMainWindow" name="inputwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>851</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>40</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Opciones de gráfica</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="xmin">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>90</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <double>-9999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
    <property name="value">
     <double>-100.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="xmax">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>90</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-9999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>90</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>De</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>90</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hasta</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFuncion">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>240</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Función modelada:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>90</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>160</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>160</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>De</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="xmin_2">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>160</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <double>-9999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
    <property name="value">
     <double>-100.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="xmax_2">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>160</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-9999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>160</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hasta</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout"/>
   </widget>
   <widget class="QLabel" name="labelClasificacion">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>300</y>
      <width>391</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>610</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar y Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>610</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
